.github/workflows/build-windows.yml

```diff
@@ -7,6 +7,7 @@ on:
   pull_request:
     branches:
       - 'main'
+      - 'fix/*'
```

Comment:
We understand this was for testing, so we'd get a CI run on this PR - given the normal flow is to PR to main, we're unsure if it should be kept or dropped now it's served its purpose in this particular situation. What are your thoughts here (and likewise for the other two workflow files)?

Reply:
That's a fair justification for having it - hadn't thought about it that way. We'll look at loosening the restrictions on CI generally in a separate PR then as you're right that this has utility and value given that context.